<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1233</width>
    <height>559</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1233</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">qlineargradient:(spread:pad, x1:0, y1:0, x2:1, y2:1, stop:0 rgba(191, 191, 191, 255), stop:1 rgba(255, 255, 255, 255))</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget_4">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>20</y>
     <width>711</width>
     <height>121</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_4">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Коэффициент передискретизации</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Kss">
          <property name="value">
           <number>4</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Дискретность по долготе, м</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dxsint">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Дискретность по широте, м</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="dysint">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_8">
        <item>
         <widget class="QLabel" name="label_7">
          <property name="text">
           <string>Степень яркостного изображения</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDoubleSpinBox" name="StepBright">
          <property name="value">
           <double>1.000000000000000</double>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QLabel" name="label_8">
          <property name="text">
           <string>Число точек по долготе</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Nxsint">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>201</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <widget class="QLabel" name="label_9">
          <property name="text">
           <string>Число точек по широте</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="Nysint">
          <property name="maximum">
           <number>1000</number>
          </property>
          <property name="value">
           <number>201</number>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QGroupBox" name="TypeWinDn">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>351</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Весовая функция по наклонной дальности</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QRadioButton" name="r1">
       <property name="text">
        <string>нет</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r2">
       <property name="text">
        <string>косинус</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r3">
       <property name="text">
        <string>косинус квадрат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r4">
       <property name="text">
        <string>Хемминга</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r5">
       <property name="text">
        <string>Хемминга (третья степень)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r6">
       <property name="text">
        <string>Хемминга (четвертая степень)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r7">
       <property name="text">
        <string>Кайзера-Бесселя, alfa=2.7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r8">
       <property name="text">
        <string>Кайзера-Бесселя, alfa=3.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r9">
       <property name="text">
        <string>Кайзера-Бесселя, alfa=3.5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="r10">
       <property name="text">
        <string>Блекмана-Херриса</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="TypeWinDp">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>150</y>
     <width>351</width>
     <height>381</height>
    </rect>
   </property>
   <property name="title">
    <string>Весовая функция по поперечной дальности</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="verticalLayoutWidget_5">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>24</y>
      <width>261</width>
      <height>361</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QRadioButton" name="p1">
       <property name="text">
        <string>нет</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p2">
       <property name="text">
        <string>косинус</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p3">
       <property name="text">
        <string>косинус квадрат</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p4">
       <property name="text">
        <string>Хемминга</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p5">
       <property name="text">
        <string>Хемминга (третья степень)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p6">
       <property name="text">
        <string>Хемминга (четвертая степень)</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p7">
       <property name="text">
        <string>Кайзера-Бесселя, alfa=2.7</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p8">
       <property name="text">
        <string>Кайзера-Бесселя, alfa=3.1</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p9">
       <property name="text">
        <string>Кайзера-Бесселя, alfa=3.5</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="p10">
       <property name="text">
        <string>Блекмана-Херриса</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_3">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>150</y>
     <width>391</width>
     <height>101</height>
    </rect>
   </property>
   <property name="title">
    <string>Режим съемки</string>
   </property>
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>19</y>
      <width>391</width>
      <height>81</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QRadioButton" name="Stripmap">
       <property name="text">
        <string>Маршрутный</string>
       </property>
       <property name="checked">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Stripmap2">
       <property name="text">
        <string>Маршрутный
 скошенный</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="Detailmode">
       <property name="text">
        <string>Детальный</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QCheckBox" name="checkBox">
   <property name="geometry">
    <rect>
     <x>930</x>
     <y>270</y>
     <width>281</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Однопроходная интерферометрия</string>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>310</y>
     <width>391</width>
     <height>119</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_6">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_11">
      <item>
       <widget class="QLabel" name="label_10">
        <property name="text">
         <string>Время синтезирования, c</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="Tsint">
        <property name="value">
         <double>0.400000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_11">
        <property name="text">
         <string>Задержка момента построения РЛИ-1, с</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="tauRli"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_13">
      <item>
       <widget class="QLabel" name="label_12">
        <property name="text">
         <string>Интервал времени между РЛИ-1,-2. с</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QDoubleSpinBox" name="t_r_w">
        <property name="value">
         <double>0.100000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="checkBox_2">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>500</y>
     <width>181</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Отображать сигналы</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Start">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>490</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Расчет РЛИ</string>
   </property>
  </widget>
  <widget class="QPushButton" name="upload">
   <property name="geometry">
    <rect>
     <x>1040</x>
     <y>110</y>
     <width>161</width>
     <height>29</height>
    </rect>
   </property>
   <property name="text">
    <string>Выбрать файл ТС</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>20</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Путь:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="Path">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>800</x>
     <y>60</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Файл ТС:</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="File_name">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QCheckBox" name="GPU">
   <property name="geometry">
    <rect>
     <x>820</x>
     <y>450</y>
     <width>271</width>
     <height>26</height>
    </rect>
   </property>
   <property name="text">
    <string>Аппаратное ускорение (GPU)</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
